Objects/obmalloc.c

```diff
@@ -1628,21 +1629,19 @@ pymalloc_alloc(void *ctx, void **ptr_p, size_t nbytes)
          */
         bp = allocate_from_new_pool(size);
         if (UNLIKELY(bp == NULL)) {
```

Comment:
It is no longer needed.

Reply:
Can you please elaborate? I didn't modify allocate_from_new_pool: it can still return NULL.

Reply:
Oh, I now understood the comment: I removed this code.